tooling/cli/src/interface/rust.rs

```diff
@@ -420,7 +433,21 @@ impl Rust {
         .unwrap_or_else(|| vec![tauri_path])
     };
 
+    let watch_folders = watch_folders
+      .iter()
```

Comment:
```suggestion
      .into_iter()
```

Should remove the need for `.to_owned` later, no?

Reply:
first time using rust.